src/main/resources/api/service_openapi.yaml

```diff
@@ -0,0 +1,36 @@
+openapi: 3.0.3
```

Comment:
nit: Is the filename standard for Terra platform? might have just expected `tanagra.yaml`